scripts/e2e/generate-e2e-spec.sh

```diff
@@ -0,0 +1,24 @@
+echo "Creating e2e spec files using 'generate-yamls.sh'..."
+
+# first set the appropriate envoironment variables gor generate-yamls.sh
+export VSPHERE_SERVER='$VSPHERE_SERVER'
+export VSPHERE_USERNAME='$VSPHERE_USERNAME'
+export VSPHERE_PASSWORD='$VSPHERE_PASSWORD'
+export VSPHERE_DATACENTER='SDDC-Datacenter'
+export VSPHERE_DATASTORE='WorkloadDatastore'
+export VSPHERE_NETWORK='sddc-cgw-network-5'
+export VSPHERE_RESOURCE_POOL='clusterapi'
+export VSPHERE_FOLDER='clusterapi'
+export VSPHERE_TEMPLATE='ubuntu-1804-kube-v1.13.6'
+
+
+# run the script to generate the yaml files
+sh ../../../hack/generate-yaml.sh -f -c 'test1' -m '$VSPHERE_CONTROLLER_VERSION' \
```

Comment:
Hi @ykakarap,

Please use docker-in-docker to ensure we're dogfooding the getting started guide. We should be using the `manifests` image to generate the content.

Reply:
Hi @ykakarap,

I think you should sync up with @codenrhoden tomorrow morning. He's doing work on the CI/e2e process, and it's possible you could use the image he's building elsewhere to generate the manifests here.

Reply:
Made the changes to use `manifests` docker image to generate the files.